app/jobs/dradis/plugins/csv/mapping_import_job.rb

```diff
@@ -0,0 +1,82 @@
+require 'csv'
+
+module Dradis::Plugins::CSV
+  class MappingImportJob < ApplicationJob
+    queue_as :dradis_project
+
+    def perform(file:, identifier_col_index:, mappings:, project_id:, uid:)
+      @logger = Log.new(uid: uid)
+      @logger.write { "Job id is #{job_id}." }
+      @logger.write { 'Worker process starting background task.' }
+
+      @file = file
+      @identifier_col_index = identifier_col_index
+      @mappings = mappings
+      @project = Project.find(project_id)
+      node_col_mapping = @mappings.find { |index, field| field['type'] == 'node' }
+      @node_col_index = node_col_mapping[0].to_i if node_col_mapping
+
+      unless @identifier_col_index
+        @logger.fatal('Unique Identifier doesn\'t exist, please choose a column as the Unique Identifier.')
+        return
+      end
+
+      import_csv!
+
+      @logger.write { 'Worker process completed.' }
+    end
+
+    private
+
+    def content_service
+      @content_service ||= Dradis::Plugins::ContentService::Base.new(
+        project: @project,
+        plugin: Dradis::Plugins::CSV
+      )
+    end
+
+    def import_csv!
+      @issue_mappings = @mappings.select { |index, mapping| mapping['type'] == 'issue' }
+      @evidence_mappings = @mappings.select { |index, mapping| mapping['type'] == 'evidence' }
```

Comment:
Alpha and I think we can drop plural here since each is now a single mapping.

```suggestion
      @evidence_mapping = @mappings.select { |index, mapping| mapping['type'] == 'evidence' }
      @issue_mapping = @mappings.select { |index, mapping| mapping['type'] == 'issue' }
```

Reply:
It's not a single mapping. 

Mappings has the follow format:
```ruby
@mappings = {
  '0' => { 'type' => 'node' },
  '1' => { 'type' => 'issue', field: 'Title' },
  '2' => { 'type' => 'issue', field: 'Description' },
  '3' => { 'type' => 'evidence', field: 'Port' },
  '4' => { 'type' => 'evidence', field: 'Location' }
}
```

`@mappings.select { |index, mapping| mapping['type'] == 'issue' }` will return only issue mappings: e.g.
```ruby
{
  '1' => { 'type' => 'issue', field: 'Title' },
  '2' => { 'type' => 'issue', field: 'Description' }
}
```

Reply:
I was thinking that this is a set of mappings:
```
@mappings = {
  '0' => { 'type' => 'node' },
  '1' => { 'type' => 'issue', field: 'Title' },
  '3' => { 'type' => 'evidence', field: 'Port' },
}
```

This is a mapping (singular) of issue fields:
```
{
  '1' => { 'type' => 'issue', field: 'Title' },
  '2' => { 'type' => 'issue', field: 'Description' }
}
```

This is a mapping (singular) of evidence fields:
```
{
  '3' => { 'type' => 'evidence', field: 'Port' },
  '4' => { 'type' => 'evidence', field: 'Location' }
}
```

This is a very minor thing so not a hill I'm willing to die on 😀

Reply:
Ah okay, I think i'm just gonna leave it because i'm already using the `mapping` var (singular) inside the blocks already.